packages/plugin-conversation/src/decryption-transforms.js

```diff
@@ -37,8 +37,15 @@ export const transforms = toArray(`inbound`, {
     }
 
     const usableKey = conversation.encryptionKeyUrl || key;
+    const decryptionFailureMessage = ctx.spark.conversation.config.decryptionFailureMessage;
+
     if (usableKey) {
-      promises.push(ctx.transform(`decryptPropDisplayName`, usableKey, conversation));
+      promises.push(ctx.transform(`decryptPropDisplayName`, usableKey, conversation)
+        .catch((error) => {
+          console.error(error.stack);
```

Comment:
if this was intended, use `this.logger`, otherwise, please remove. also, log a helpful message instead of just the stack